org.civicrm.bootstrapcivicrm/scss/civicrm/common/_tables.scss

```diff
@@ -113,6 +113,169 @@
   }
 }
 
+%expandable-table {
```

Comment:
Do we *really* need an `@extend`? Is it absolutely necessary? It is an extremely dangerous directive to use, and we're still paying the price for it. Couldn't yo instead do a mixin?